src/index.tsx

```diff
@@ -210,6 +217,33 @@ function invariant(value: any, message?: string) {
   }
 }
 
+function assignRef<RefValueType = any>(
```

Comment:
We should probably use `unknown` instead
https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html#any

```suggestion
function assignRef<RefValueType = unknown>(
```